log_entries.py

```diff
@@ -9,12 +9,6 @@
 class LogEntry(dict):
     """ Provides comparison operators to the dictionary elements
     """
-    def __eq__(self, other):
```

Comment:
this override was preventing assertDictEqual from checking all dict keys in unit test, the only comparison being done here is lt